tools/pepsi/src/pre_game.rs

```diff
@@ -37,9 +37,10 @@ pub struct Arguments {
     /// Skip the OS version check
     #[arg(long)]
     pub skip_os_check: bool,
-    /// Cycler instances to record e.g. Control or VisionBottom (don't specify to disable recording)
-    #[arg(long, default_value = "Control")]
-    pub cycler_instances_to_be_recorded: Vec<String>,
+    /// Intervals between cycle recordings, e.g. Control=1,VisionTop=30 to record every cycle in Control
+    /// and one out of every 30 in VisionTop. Set to 0 or don't specify to disable recording for a cycler.
+    #[arg(long, value_delimiter=',', value_parser = parse_key_value::<String, usize>, default_value = "Control=1,VisionTop=30,VisionBottom=30")]
+    pub recording_intervals: Vec<(String, usize)>,
```

Comment:
Is it possible to change the type to `HashMap<String, usize>`? This way you could drop the conversion from `Vec` to `HashMap`?

Reply:
`Vec` is natively supported by `clap_derive`, while `HashMap` is unfortunately not. By using a `Vec`, only the tuples themselves instead of the whole collection has to be parsed, otherwise we would also need to add a parser for the `HashMap` itself.